values.yaml

```diff
@@ -91,8 +93,8 @@ syncer:
     firmware_manifest:
       url: http://modeldata/modeldata.json
 
-fleet_scheduler:
-  enable: false # when enabled, metal-toolbox/fleet-scheduler will need to be deployed with `make push-image-devel`
+fleet-scheduler:
+  enable: true # when enabled, metal-toolbox/fleet-scheduler will need to be deployed with `make push-image-devel`
```

Comment:
How would you test local changes to fleet-scheduler without going thru the upstream repo?

Reply:
I will add some notes to inform people how to do that.